lib/vanagon/platform/windows.rb

```diff
@@ -53,9 +53,21 @@ def generate_packaging_artifacts(workdir, name, binding)
       # @param name [String] name of the project
       # @param binding [Binding] binding to use in evaluating the packaging templates
       def generate_msi_packaging_artifacts(workdir, name, binding)
-        FileUtils.mkdir_p(File.join(workdir, "wix"))
-        erb_file(File.join(VANAGON_ROOT, "resources/windows/wix/project.wxs.erb"), File.join(workdir, "wix",  "#{name}.wxs"), false, { :binding => binding })
-        erb_file(File.join(VANAGON_ROOT, "resources/windows/wix/project.filter.xslt.erb"), File.join(workdir, "wix", "#{name}.filter.xslt"), false, { :binding => binding })
+        # Copy (cp -r) files to workdir/wix folder. The VANAGON_ROOT root files are copied first, followed by the project specific files.
+        # The "remove_destination" flag is set which ensures that project specific files will always over-write generic/vanagon files
+        FileUtils.cp_r(File.join(VANAGON_ROOT, "resources/windows/wix"), workdir, :verbose => true, :remove_destination => true)
+        FileUtils.cp_r("resources/windows/wix", workdir, :verbose => true, :remove_destination => true)
+        # Evaluate and process all *.erb files found in workdir/wix
+        # The originating .erb file is left in place as it is ignored by the wix/candle process.
+        Dir.chdir(File.join(workdir, "wix")) do
+          Dir.glob("**/*.erb").each do |erb_filename|
+            target_file = File.basename(erb_filename, ".erb")
+            if File.exists?(target_file)
+              fail "Template file #{erb_filename} conflicts with static file #{target_file}"
```

Comment:
You can wrap the use of `cp` within a function (you probably don't need recursion since you're copying files). I'm going to write some examples for this for you.